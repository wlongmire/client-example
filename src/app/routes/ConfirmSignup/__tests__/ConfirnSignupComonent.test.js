/* eslint-env node, jest */

import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { ConfirmSignup } from '../index'

jest.mock('react-router', () => ({ browserHistory: [] }))

describe('>>> ConfirmSignup Component Snapshot', () => {
  it('+++capturing Snapshot with', () => {
    const renderedValue = shallow(
      <ConfirmSignup
        dispatch={jest.fn()}
        location={{ query: { key: '12123123123' } }}
      />)
    expect(toJson(renderedValue)).toMatchSnapshot()
  })
})