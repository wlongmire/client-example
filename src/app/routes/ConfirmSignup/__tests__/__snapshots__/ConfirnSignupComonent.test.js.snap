// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> ConfirmSignup Component Snapshot +++capturing Snapshot with 1`] = `
<div
  className="signupBackground"
>
  <div
    className="signupHeader"
  >
    <h1>
      Your Owner's Edge account
    </h1>
    <h3 />
  </div>
  <div
    className="currentHeader"
  >
    <SignupHeader
      header1="ACCOUNT STEP 1 OF 3"
      header2="Choose a new password"
    />
  </div>
  <div
    className="currentStep"
  >
    <Connect(SetPassword)
      cognitoUser={null}
      goToNextStep={[Function]}
      urlKey="12123123123"
    />
  </div>
</div>
`;
